src/QualificationRunner.Core/OSPSuiteCore/QualificationConfiguration.cs

```diff
@@ -1,8 +1,21 @@
-﻿using OSPSuite.Core.Domain;
+﻿using System.Collections.Generic;
+using System.Linq;
+using OSPSuite.Core.Domain;
 using OSPSuite.Utility.Validation;
 
 namespace OSPSuite.Core.Qualification
 {
+   public interface IReferencingProject

```

Comment:
I'll reorganize those guys when I move the object to core. For now only one file for simplicity